pandas/tests/test_graphics.py

```diff
@@ -183,7 +193,7 @@ def _check_colors(self, collections, linecolors=None, facecolors=None,
         """
 
         from matplotlib.lines import Line2D
-        from matplotlib.collections import Collection
+        from matplotlib.collections import Collection, PolyCollection
```

Comment:
@TomAugspurger as an aside
if it makes things easier now (or in a later version )
we could move up the min supported version of matplotlib from 1.2
cc @tacaswell